### PR TITLE
[doc](explode-json-object)add explode-json-object usage

### PR DESCRIPTION
**Pull request by amorynan**
add doc for new table func: explode_json_func
link pr:  https://github.com/apache/doris/pull/36887

### CODE REVIEW

File: sidebars.json [hunk 1]
```diff
@@ -1067,7 +1067,8 @@
                                 "sql-manual/sql-functions/table-functions/jobs",
                                 "sql-manual/sql-functions/table-functions/mv_infos",
                                 "sql-manual/sql-functions/table-functions/tasks",
-                                "sql-manual/sql-functions/table-functions/query"
+                                "sql-manual/sql-functions/table-functions/query",
+                                "sql-manual/sql-functions/table-functions/explode_json_object"
```
Comment (luzhijing) [inline sidebars.json:1071]: explode-json-object

File: docs/sql-manual/sql-functions/table-functions/explode-json-object.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+KIND, either express or implied. See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## EXPLODE_JSON_OBJECT
+
+### Description
+
+Expand a json object. `explode_json_object_outer` will return NULL, while given json is NULL or is not a json object.
```
Comment (luzhijing) [inline docs/sql-manual/sql-functions/table-functions/explode-json-object.md:31]: json -> JSON

File: docs/sql-manual/sql-functions/table-functions/explode-json-object.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+> Note.
```
Comment (luzhijing) [inline docs/sql-manual/sql-functions/table-functions/explode-json-object.md:33]: :::tip Note
xxxx
:::

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/sql-manual/sql-functions/table-functions/explode-json-object.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+KIND, either express or implied. See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## EXPLODE_JSON_OBJECT
+
+### Description
+
+展开一个 json 对象。当给定的 json 为 NULL 或不是 json 对象时，`explode_json_object_outer` 将返回 NULL。
```
Comment (luzhijing) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/sql-manual/sql-functions/table-functions/explode-json-object.md:31]: JSON

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/sql-manual/sql-functions/table-functions/explode-json-object.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+
+> 注意:
+>
+> 表函数必须与later view 结合使用
```
Comment (luzhijing) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/sql-manual/sql-functions/table-functions/explode-json-object.md:35]: LATERAL VIEW

**Comment by amorynan** [comment]
run buildall

**Review by luzhijing — APPROVED** [review]
LGTM

**Review by shasevendow — APPROVED** [review]
LGTM

File: docs/sql-manual/sql-functions/table-functions/explode-json-object.md
Comment (KassieZ) [inline docs/sql-manual/sql-functions/table-functions/explode-json-object.md:1]: Should also update verison docs if the functions work on v3.0 / 2.1 / 2.0

**Comment by KassieZ** [comment]
Thank you for your contribution. However, this pull request contains comments and unresolved conflicts. As a result, it will be closed. If you still wish to submit your changes, please create a new pull request.

**Pull request closed without merging**